Tchap/Modules/Home/HomeCoordinator.swift

```diff
@@ -1,544 +1,544 @@
-                    message = TchapL10n.inviteAlreadySentByEmail(email)
-                case .inviteIgnoredForDiscoveredUser(userID: let userID):
-                    discoveredUserID = userID
-                    message = TchapL10n.inviteNotSentForDiscoveredUser
-                case .inviteIgnoredForUnauthorizedEmail:
-                    message = TchapL10n.inviteNotSentForUnauthorizedEmail(email)
-                }
-                
-                sself.currentAlertController?.dismiss(animated: false)
-                
-                let alert = UIAlertController(title: TchapL10n.inviteInformationTitle, message: message, preferredStyle: .alert)
-                
-                let okTitle = Bundle.mxk_localizedString(forKey: "ok")
-                let okAction = UIAlertAction(title: okTitle, style: .default, handler: { action in
-                    if let userID = discoveredUserID {
-                        // Open the discussion
-                        sself.startDiscussion(with: userID)
-                    }
-                })
-                alert.addAction(okAction)
-                sself.currentAlertController = alert
-                
-                homeViewController.present(alert, animated: true, completion: nil)
-            case .failure(let error):
-                let errorPresentable = sself.inviteErrorPresentable(from: error)
-                sself.errorPresenter?.present(errorPresentable: errorPresentable, animated: true)
-            }
-        }
-        
-    }
-    
-    private func inviteErrorPresentable(from error: Error) -> ErrorPresentable {
-        let errorTitle = TchapL10n.inviteSendingFailedTitle
-        let errorMessage: String
-        
-        let nsError = error as NSError
-        
-        if let message = nsError.userInfo[NSLocalizedDescriptionKey] as? String {
-            errorMessage = message
-        } else {
-            errorMessage = TchapL10n.errorMessageDefault
-        }
-        
-        return ErrorPresentableImpl(title: errorTitle, message: errorMessage)
-    }
-}
-
-// MARK: - SettingsCoordinatorDelegate
-extension HomeCoordinator: SettingsCoordinatorDelegate {
-    func settingsCoordinator(_ coordinator: SettingsCoordinatorType, reloadMatrixSessionsByClearingCache clearCache: Bool) {
-        self.navigationRouter.popToRootModule(animated: false)
-        self.delegate?.homeCoordinator(self, reloadMatrixSessionsByClearingCache: clearCache)
-    }
-}
-
-// MARK: - GlobalSearchBarDelegate
-extension HomeCoordinator: GlobalSearchBarDelegate {
-    func globalSearchBar(_ globalSearchBar: GlobalSearchBar, textDidChange searchText: String?) {
-//        self.roomsCoordinator?.updateSearchText(searchText)
-        self.contactsCoordinator?.updateSearchText(searchText)
-    }
-}
-
-// MARK: - RoomsCoordinatorDelegate
-//extension HomeCoordinator: RoomsCoordinatorDelegate {
-//    func roomsCoordinator(_ coordinator: RoomsCoordinatorType, didSelectRoomID roomID: String) {
+///*
```

Comment:
Could we ignore this file in project instead of comment it ?

Reply:
HomeCoordinator is not used anymore in the project, but still available in the repo. We should clean all the unused files when the re-alignment onto Element will be all finished (to avoid the lost of some Tchap code / use cases).

Reply:
You should restore this file and ignore it in the project